src/continual/multimodal_trainer.py

```
@@ -0,0 +1,510 @@
+            self.fusion_layer = VisionTextFusion(fusion_config).to(
+                self.device
+            )
+
+        # Track vision domains
+        self.vision_domains = set(config.vision_domains)
+        self.active_vision_domain = adapter_name
+
+    def add_domain_adapter(
+        self,
+        domain_name: str,
+        vision_only: bool = False
```

Ruff (0.14.2)

Unused method argument: `vision_only`

(ARG002)